backend/src/main/java/com/github/bolabolor/backend/security/MongoUsersDetailsService.java

```diff
@@ -0,0 +1,22 @@
+package com.github.bolabolor.backend.security;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.security.core.userdetails.User;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.stereotype.Service;
+import java.util.Collections;
+@Service
+@RequiredArgsConstructor
+public class MongoUsersDetailsService implements UserDetailsService {
+    private final MongoUserRepository mongoUserRepository;
+
+    @Override
+    public UserDetails loadUserByUsername(String username) throws UsernameNotFoundException{
+        MongoUser mongoUser=mongoUserRepository.findMongoUserByUsername(username)
+        .orElseThrow(() -> new UsernameNotFoundException("User with name: " + username + " not found!"));
```

Comment:
Einmal cmd + opt + L drücken, dann hast du Autoformat. Ansonsten zur besseren Lesbarkeit den Code einrücken:
```suggestion
        MongoUser mongoUser = mongoUserRepository
            .findMongoUserByUsername(username)
            .orElseThrow(() -> new UsernameNotFoundException("User with name: " + username + " not found!"));
```